toolbox/incari/object/set-alpha.md

```diff
@@ -0,0 +1,34 @@
+# Overview
+
+![The Set Alpha Node.](../../../.gitbook/assets/toolbox/incari/object/set-alpha.PNG)
+
+**Set Alpha** sets the `alpha` component value of the color space of an **Object**.
+
+The **Set Alpha** node only works with objects that have **Sprite** category as an attribute such as **List**, **Image**, **Sequence**, **Web Sprite**, **Sprite** and **Text**.
+
+# Attributes
+
+|Attribute|Type|Description|
+|---|---|---|
+|`Object`|**ObjectID**|The **Object** whose color space's `alpha` component is to be set, if one is not provided in the `object ID` **Socket**.|
+|`Default Alpha`|**Float**|The default value of the **Object** color space's `alpha` component. 
```

Comment:
The value of the **Object**'s `Alpha` **Attribute** if one isn't provided in the `Object ID` **Socket** .